api/src/application/http/role/handlers/get_role.rs

```diff
@@ -34,8 +34,7 @@ pub struct GetRoleResponse {
     )
 )]
 pub async fn get_role(
-    Path(realm_name): Path<String>,
-    Path(role_id): Path<Uuid>,
+    Path((realm_name, role_id)): Path<(String, Uuid)>,
```

Comment:
I'm using 0.8.4 axum, and these changes were necessary for me to avoid a panic, namely
```
 Note that multiple parameters must be extracted with a tuple `Path<(_, _)>` or a struct `Path<YourParams>
 ```